crates/burn-import/src/burn/node/binary.rs

```diff
@@ -131,6 +131,9 @@ impl BinaryNode {
             (Type::Tensor(_), Type::Tensor(_)) => move |lhs, rhs| quote! { #lhs.sub(#rhs) },
             (Type::Tensor(_), Type::Scalar(_)) => move |lhs, rhs| quote! { #lhs.sub_scalar(#rhs) },
             (Type::Scalar(_), Type::Scalar(_)) => move |lhs, rhs| quote! { #lhs - #rhs },
+            (Type::Scalar(_), Type::Tensor(_)) => {
+                move |lhs, rhs| quote! { #rhs.mul_scalar(-1).add_scalar(#lhs) }
```

Comment:
I think it would be more efficient if we have one tensor op and rely on compiler to negate `#lhs`. We can rewrite as follows: `#rhs.add_scalar(-(#lhs)). So generated code might look like this: `#rhs.add_scalar(-(- 42)). And Rust compiler will precompute number literal correctly.

Reply:
Thanks or review @antimora!

Actually since this is lhs (scalar) - rhs (tensor), looks like -#rhs.sub_scalar(#lhs) produced the right result. I will make an update.